src/utils/daos_autotest.c

```diff
@@ -727,12 +767,13 @@ kv_punch(void)
 		return rc;
 	}
 
-	punch_rc = daos_obj_punch(oh, DAOS_TX_NONE, 0, NULL);
+	punch_rc = daos_obj_punch(daos_kv2objhandle(oh), DAOS_TX_NONE,
```

Comment:
Is it guaranteed that the API will gracefully handle DAOS_HDL_INVAL return from daos_kv2objhandle() ?

Reply:
Agreed, bttter add check, and addressed above johan's comments as well.